dgm-builder/src/main/resources/com/cloudbees/groovy/cps/tool/translatable.txt

```diff
@@ -1,133 +1,88 @@
-org.codehaus.groovy.runtime.DefaultGroovyMethods.<E,T,U,V>inject(E[],U,groovy.lang.Closure<V>)
```

Comment:
Just resorting here? Would rather keep the original sort order to minimize diff and show that you are only adding entries.